repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/guidance-content.ftl

```diff
@@ -52,7 +52,17 @@
                         </section>
                     </div>
 
-                    <@hee.quickLinks quickLinks=guidanceDocument.quickLinks/>
+                    <#if guidanceDocument.contentBlocks??>
```

Comment:
Should this suppose to check `guidanceDocument.rightHandBlocks` ?

Reply:
@smanivasag Thank for the noticed. I updated that

Reply:
Thanks @thojava 👍